superset/models/dashboard.py

```diff
@@ -179,6 +179,33 @@ def dashboard_link(self) -> Markup:
         title = escape(self.dashboard_title or "<empty>")
         return Markup(f'<a href="{self.url}">{title}</a>')
 
+    @property
+    def dashboard_title_raw(self) -> str:
+        return self.dashboard_title
+
+    @property
+    def json(self):
+        return self.changed_by.to_json()
```

Comment:
The output from this includes an hashed password, you can selectively add related fields using dotted notation on `list_columns`, ex:

```
    list_columns = [
        "id",
        "dashboard_title",
        "url",
        "published",
        "owners_json",
        "changed_by.username",
        "changed_by_name",
        "changed_by_url",
        "changed_on",
    ]
```